app/views/fact/doi.html.erb

```diff
@@ -0,0 +1,19 @@
+<% return unless @json.present? %>
+
+<div id="doi-fact" class="fact">
```

Comment:
It looks like we have repeated id values for this structure, between the search partials and the fact templates? Can we remove one, or should we change them to not be identical?

(I'm making the same comment on each of the `doi-fact` elements, but I think all four are doubled)